styles/navigation.css

```diff
@@ -0,0 +1,43 @@
+.navbar-wrapper {
+  padding: var(--spacing-lg);
+  background: var(--white-color);
+  display: flex;
+  justify-content: space-between;
+  gap: var(--spacing-md);
+}
+
+.navbar-action-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: var(--spacing-md);
+}
+
+.nav-cta-btn {
+  display: flex;
+  justify-content: space-between;
+  gap: var(--spacing-md);
+}
+
+
+@media (max-width: 1050px) {
+    .nav-wrapper {
+        padding: 0 1rem;
```

Comment:
you could have used padding-inline instead of padding if you only needed left right padding